docs/CVE-2024-11053.md

```diff
@@ -62,6 +62,14 @@ SOLUTION
 The fix also addresses a few other .netrc related issues. The fix caused a
 regression that was fixed by the second commit `9fce2c5`.
 
+### 2025-09-15 addendum
```

Comment:
I suggest you instead use an explanatory subtitle like:

`### For curl versions before 8.11.0` or similar